restapi/endpoints_test.go

```diff
@@ -345,6 +345,24 @@ func (suite *endpointTestSuite) TestGetAllStories() {
 	response.Object().Value("payload").Array().Element(1).Object().Value("current_city").Equal("Toronto")
 	response.Object().Value("payload").Array().Element(1).Object().Value("tags").Array().Empty()
 }
+
+func (suite *endpointTestSuite) TestGetVisibleCountries() {
```

Comment:
I'd like to see a test to make sure that no duplicate countries are returned.

Reply:
I think this test covers it because there are two stories with the Country as France, but only one of them shows up in the returned array.